.github/workflows/ci.yml

```diff
@@ -576,7 +576,7 @@ jobs:
       - run: dart pub get
 
       - name: Deploy
-        run: dart run grinder pkg-chocolatey-deploy
+        run: dart run grinder protobuf pkg-chocolatey-deploy
```

Comment:
I think we need to run `./.github/util/initialize`. Otherwise we'll be missing `buf`.